pkg/badgerdb/storage.go

```diff
@@ -1,11 +1,26 @@
-// storage.go
 package badgerdb
```

Comment:
I see why this change makes more sense than how I had it under `main` @restevens402 ✅